src/system/CategoriesModule/Controller/ConfigController.php

```diff
@@ -50,7 +50,8 @@ public function configAction(Request $request)
 
         $form = $this->createForm('Zikula\CategoriesModule\Form\Type\ConfigType',
             $modVars, [
-                'translator' => $this->get('translator.default')
+                'translator' => $this->get('translator.default'),
+                'request' => $request
```

Comment:
if the only thing that is needed is the locale from the request then only send the locale